validate.php

```diff
@@ -96,11 +84,36 @@ function validatePlugin(SplFileInfo $pluginDirectory): int
     return $errors;
 }
 
+function checkPluginsJson(): int
+{
+    ob_start();
+    include __DIR__ . '/generate-json.php';
+    $correctJson = ob_get_clean();
+
+    $currentJson = file_get_contents(__DIR__ . '/plugins.json');
+
+    if (json_decode($currentJson, true) === json_decode($correctJson, true)) {
```

Comment:
do you really need to decode? or even compare whole jsons? I think file hash would be enough

Reply:
I'm worried about errors caused by LF vs CRLF.